src/Slider.jsx

```diff
@@ -94,6 +94,21 @@ class Slider extends React.Component {
     this.onChange({ value });
   }
 
+  onKeyboard(e) {
+    const valueMutator = utils.getKeyboardValueMutator(e);
+
+    if (valueMutator) {
+      utils.pauseEvent(e);
+      const state = this.state;
+      const oldValue = state.value;
```

Comment:
Haha, good to know! Let's keep this for now unchanged. If I have some time again after this PR, I will try to create one for some refactoring 😄